.github/workflows/test.yml

```diff
@@ -1,21 +1,11 @@
-name: Python and PHP Tests
+name: Tests
```

Comment:
Maybe call this `Lint` and the file `test-lint.yml` because it doesn't actually run any code tests

Reply:
I don't know, I expect to run pylint somewhere at some point and it would obviously go into lint as the name suggests it, but we only want to run linting on python code changes. So naming is not perfect yet.

Reply:
The exclude paths can be used for a specific job as far as I know